common/globalTruffleConfig.js

```diff
@@ -21,7 +21,7 @@ const infuraApiKey = process.env.INFURA_API_KEY ? process.env.INFURA_API_KEY : "
 
 // Default options
 const gasPx = 20000000000; // 20 gwei
-const gas = 6720000; // Conservative estimate of the block gas limit.
+const gas = 9000000; // Conservative estimate of the block gas limit.
```

Comment:
This I updated to be more realistic. Current gass limit is ~10 million gas per block. This has been the average for about[ 6 months](https://etherscan.io/chart/gaslimit). making this 9 million gives us the wiggle room we need to still run tests.